src/libappimage_shared/elf.c

```diff
@@ -117,6 +117,41 @@ static off_t read_elf64(FILE* fd)
 	return sht_end > last_section_end ? sht_end : last_section_end;
 }
 
+ssize_t appimage_get_elf_size(const char* fname) {
```

Comment:
Hm, not really a fan of reimplementing this although we have `appimage_get_payload_offset`... need to think about this.